### ISSUE TITLE
Stopped working with PhpStorm terminal in latest version of Fig

### ISSUE THREAD

**Issue by JasonTheAdams** [withfig/fig#1992]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.0 (22A380)

### Expected behaviour

Fig has worked well with PhpStorm (and other IntelliJ IDE's) for a while now. I recently updated to the latest version of Fig and it simply doesn't work anymore in the PhpStorm terminal.

### Actual behaviour

Well, I would expect it to pop up when I type. I've also noticed that IntelliJ doesn't even show up in the Integrations list in the Fig dashboard settings??	

### Steps to reproduce

It just doesn't run. Not sure what steps to include.

### Environment

```yaml
fig-details:
  - 2.7.4
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.0.0 (22A380)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/jasonadams
  - exe-path: /usr/local/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 614134a4-43e4-4d48-bbf8-52e9f1d849b9
    - FIG_HOSTNAME: Jasons-MBP16.local
    - FIG_TERM: 2.7.4
    - PATH: /usr/local/opt/mysql-client@5.7/bin:/usr/local/opt/ruby/bin:~/.composer/vendor/bin:/Users/jasonadams/.nvm/versions/node/v14.19.2/bin:/Library/Frameworks/Python.framework/Versions/3.9/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/jasonadams/.fig/bin:/Users/jasonadams/.local/bin:/Users/jasonadams/.composer/vendor/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PID: 15693
    - FIG_PARENT: 614134a4-43e4-4d48-bbf8-52e9f1d849b9
    - FIG_SET_PARENT: 614134a4-43e4-4d48-bbf8-52e9f1d849b9
```

**Comment by mschrage** [comment]
Try running `fig doctor` in the PhpStorm terminal?

**Comment by JasonTheAdams** [comment]
Said there were no problems and that I should make an issue.

**Comment by mschrage** [comment]
How did you install PhpStorm? Did you use JetBrain toolbox by chance?

If so can you run, `echo $__CFBundleIdentifier` in the integrated terminal.

**Comment by squili** [comment]
@JasonTheAdams Could you try running `fig integrations uninstall intellij` and then `fig integrations install intellij`? You may have an old version of the integrations installed.

**Comment by JasonTheAdams** [comment]
Hi @squili, I'm afraid that didn't seem to make a difference.

Out of curiosity, why doesn't IntelliJ show up as an integration?
<img width="1142" alt="image" src="https://user-images.githubusercontent.com/2024145/200414244-a919bbd0-f2c1-4808-a5ef-fd88046e6cfe.png">

**Comment by squili** [comment]
This page is definitely the source of a lot of confusion, we'll make sure to clarify it! It should be enabled by default - what's more confusing to me is why it's not showing up at all. Maybe try running `fig integrations install input-method`?

**Comment by JasonTheAdams** [comment]
Still no good I'm afraid. It's working well in iTerm and other places, just not in PhpStorm.

**Comment by koppieesq** [comment]
I'm having the same problem.  When I try to reinstall intellij, I get an error:

```
error: IO Error: No such file or directory (os error 2)
```

I'm on MacOS Ventura with intellij WebStorm, installed through the JetBrains toolbox.  Let me know if there's anything I can do to help troubleshoot.

**Comment by squili** [comment]
@JasonTheAdams Could you try Alacritty or Kitty?
@koppieesq Could you run `FIG_LOG_LEVEL=debug FIG_LOG_STDOUT=true RUST_BACKTRACE=1 fig integrations install jetbrains` and post the output?

**Comment by koppieesq** [comment]
@squili happy to help:

```
2022-11-07T23:39:28.105842Z DEBUG fig_cli::cli: 275: Command ran: ["fig", "integrations", "install", "jetbrains"]
error: 
   0: IO Error: No such file or directory (os error 2)
   1: No such file or directory (os error 2)

Location:
   fig_cli/src/cli/integrations.rs:172

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
                                ⋮ 3 frames hidden ⋮                               
   4: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::h650b2b86a9d63a1e
      at <unknown source file>:<unknown line>
   5: fig_cli::cli::Cli::execute::{{closure}}::h9bfe2b5d435bd987
      at <unknown source file>:<unknown line>
   6: <tokio::future::poll_fn::PollFn<F> as core::future::future::Future>::poll::h48f24df4b27a07ad
      at <unknown source file>:<unknown line>
   7: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::h88bb6182b12db03a
      at <unknown source file>:<unknown line>
   8: std::thread::local::LocalKey<T>::with::h933110c7f60085b1
      at <unknown source file>:<unknown line>
   9: tokio::runtime::park::CachedParkThread::block_on::h102bd7d92a418281
      at <unknown source file>:<unknown line>
  10: tokio::runtime::scheduler::multi_thread::MultiThread::block_on::hf10794b02205ab8c
      at <unknown source file>:<unknown line>
  11: tokio::runtime::runtime::Runtime::block_on::h5b5ebb968c119e7e
      at <unknown source file>:<unknown line>
  12: fig_cli::main::h473f77c2b3446aaf
      at <unknown source file>:<unknown line>
  13: std::sys_common::backtrace::__rust_begin_short_backtrace::h16609a0ae9c747df
      at <unknown source file>:<unknown line>
  14: std::rt::lang_start::{{closure}}::hdf003a172f879abd
      at <unknown source file>:<unknown line>
  15: std::rt::lang_start_internal::hef2161f9571a51d7
      at <unknown source file>:<unknown line>
  16: _main<unknown>
      at <unknown source file>:<unknown line>

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
Run with RUST_BACKTRACE=full to include source snippets.
```

**Comment by squili** [comment]
@koppieesq How about `FIG_LOG_LEVEL=debug FIG_LOG_STDOUT=true RUST_LIB_BACKTRACE=1 fig integrations install jetbrains`?

**Comment by koppieesq** [comment]
@squili same result, unfortunately:

```
2022-11-08T00:20:02.545876Z DEBUG fig_cli::cli: 275: Command ran: ["fig", "integrations", "install", "jetbrains"]
error: 
   0: IO Error: No such file or directory (os error 2)
   1: No such file or directory (os error 2)

Location:
   fig_cli/src/cli/integrations.rs:172

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
                                ⋮ 3 frames hidden ⋮                               
   4: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::h650b2b86a9d63a1e
      at <unknown source file>:<unknown line>
   5: fig_cli::cli::Cli::execute::{{closure}}::h9bfe2b5d435bd987
      at <unknown source file>:<unknown line>
   6: <tokio::future::poll_fn::PollFn<F> as core::future::future::Future>::poll::h48f24df4b27a07ad
      at <unknown source file>:<unknown line>
   7: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::h88bb6182b12db03a
      at <unknown source file>:<unknown line>
   8: std::thread::local::LocalKey<T>::with::h933110c7f60085b1
      at <unknown source file>:<unknown line>
   9: tokio::runtime::park::CachedParkThread::block_on::h102bd7d92a418281
      at <unknown source file>:<unknown line>
  10: tokio::runtime::scheduler::multi_thread::MultiThread::block_on::hf10794b02205ab8c
      at <unknown source file>:<unknown line>
  11: tokio::runtime::runtime::Runtime::block_on::h5b5ebb968c119e7e
      at <unknown source file>:<unknown line>
  12: fig_cli::main::h473f77c2b3446aaf
      at <unknown source file>:<unknown line>
  13: std::sys_common::backtrace::__rust_begin_short_backtrace::h16609a0ae9c747df
      at <unknown source file>:<unknown line>
  14: std::rt::lang_start::{{closure}}::hdf003a172f879abd
      at <unknown source file>:<unknown line>
  15: std::rt::lang_start_internal::hef2161f9571a51d7
      at <unknown source file>:<unknown line>
  16: _main<unknown>
      at <unknown source file>:<unknown line>

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
Run with RUST_BACKTRACE=full to include source snippets.
```

**Comment by JasonTheAdams** [comment]
@squili I just installed Alacritty and Fig is working in it (albeit a bit strangely):

<img width="389" alt="image" src="https://user-images.githubusercontent.com/2024145/200467294-4171aa88-c332-4b94-a30b-9f2823734ced.png">

**Comment by mschrage** [comment]
@JasonTheAdams the issue with Alacritty is actually the result of unrelated upstream changes to input method support in `winit`.

We have a fix on beta (`fig settings app.beta true`) but you might need to restart your computer for it to kick in, due to OS-level caching.

**Comment by JasonTheAdams** [comment]
@mschrage I appreciate it, but I don't actually care about Alacritty. I was just told to test it out. My real concern here is getting things working with PhpStorm.

**Comment by JasonTheAdams** [comment]
I just updated to the latest version of Fig and it's working again! 🎉

**Comment by mschrage** [comment]
Awesome to hear @JasonTheAdams! Thanks for bearing with us.

**Comment by koppieesq** [comment]
Still not working for me.  😭  Here's what I've tried:

1. Reinstall Fig manually from the website
2. Restarted my computer
3. Upgraded to the beta version of Fig
4. `fig integrations reinstall intellij` - gave errors because of missing `plugin` directories for PhpStorm and Rider.  I added the directories and now the command succeeds - but Fig still won't work
5. `fig doctor`: "Everything looks good!"
